src/Applications/conf.yaml

```diff
@@ -14,7 +14,7 @@
 # threshold           [-r <rt>]   float     overload relative threshold
 # time_sampler_type   [-t <ts>]   str       object times sampler: <ts> in {uniform,lognormal}
 # weight_sampler_type [-w <ws>]   str       communications weights sampler: <cs> in {uniform,lognormal}
-# time_step           [-s <ts>]   int       time stepping for reading VT load logs
+# phase_id           [-s <ts>]   int       time stepping for reading VT load logs
```

Comment:
This comment should be updated to reflect the change from timestep to phase.

Reply:
Thanks, for catching that up Nicole!